src/sentry/logging/handlers.py

```diff
@@ -45,9 +33,38 @@
 )
 
 
+def _json_encoder(skipkeys: bool = False) -> json.JSONEncoder:
```

Comment:
the `*` was intentional in my github comment -- otherwise this is a [boolean trap](https://youtu.be/CnRkXO_a5mI)

Reply:
Interesting...